src/key_defs.h

```diff
@@ -16,8 +16,9 @@
 #include "KeyboardioHID.h"
 #endif
 
+namespace kaleidoscope {
 
-typedef union Key_ {
```

Comment:
In C we have to add a typedef to be able to use a struct name without prepending it with the keyword `struct`. This typedef is useless in C++.

Reply:
The original name `Key_` is preserved by an appropriate typedef in global namespace (see below).